tests/pytests/test_tags.py

```diff
@@ -162,6 +162,7 @@ def testIssue1305(env):
     env.assertEqual(res, expectedRes)
 
 def testTagCaseSensitive(env):
+    env.skipOnCluster()
```

Comment:
We shouldn't skip it, just transform env.expect to execute_command.

Reply:
We need to skip b/c we use `FT.DEBUG` which isn't supported.